src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import model.Bulldozer;
+import model.Excavator;
+import model.Truck;
+
+public class Main {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
+        truck.doWork();
+        excavator.doWork();
+        bulldozer.doWork();
+        truck.stopWork();
+        excavator.stopWork();
+        bulldozer.stopWork();
```

Comment:
here better to create an array of Machines with Truck, Bulldozer and Excavator and call methods doWork and stopWork in a loop